src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js

```diff
@@ -242,6 +242,16 @@ define([
             startSelectionOverlay.call(this);
         },
 
+        getItems: function() {
```

Comment:
Rename to `getItemIds`?

Reply:
but the function returns items ;)

Reply:
But not complete items, so this name is really confusing. Also open for other better suggestions.

Reply:
it returns the complete items when the complete items are loaded.